src/EnergyPlus/OutsideEnergySources.cc

```diff
@@ -248,8 +248,10 @@ void GetOutsideEnergySourcesInput(EnergyPlusData &state)
                                   format("{}=\"{}\" was not found.", state.dataIPShortCut->cAlphaFieldNames(4), state.dataIPShortCut->cAlphaArgs(4)));
                 ErrorsFound = true;
             }
-            if (!ScheduleManager::CheckScheduleValueMinMax(
-                    state, state.dataOutsideEnergySrcs->EnergySource(EnergySourceNum).CapFractionSchedNum, ">=", 0.0)) {
+            if (!ScheduleManager::CheckScheduleValueMinMax(state,
```

Comment:
Got rid of the string `">="` argument that was used for this function, sometimes incorrectly.  The only thing the function needs to know is if it is "greater than", or "greater than or equals".   A simple bool variable to turn it onto >= is sufficient.  No more weird string.